pennylane/interfaces/execution.py

```diff
@@ -438,30 +438,34 @@ def cost_fn(params, x):
     try:
         if mapped_interface == "autograd":
             from .autograd import execute as _execute
+
         elif mapped_interface == "tf":
             # TODO: remove pragmas when TF is supported
             import tensorflow as tf  # pragma: no cover
 
             if not tf.executing_eagerly() or "autograph" in interface:  # pragma: no cover
                 from .tensorflow_autograph import execute as _execute  # pragma: no cover
+
             else:
                 from .tensorflow import execute as _execute  # pragma: no cover
+
         elif mapped_interface == "torch":
             # TODO: remove pragmas when Torch is supported
             from .torch import execute as _execute  # pragma: no cover
-        else:  # is jax
-            # TODO: remove pragmas when Jax is supported
-            _execute = _get_jax_execute_fn(interface, tapes)  # pragma: no cover
+
+        elif mapped_interface == "jax":
+            _execute = _get_jax_execute_fn(interface, tapes)
+
+        res = _execute(
+            tapes, device, execute_fn, gradient_fn, gradient_kwargs, _n=1, max_diff=max_diff

```

Comment:
Mode is a unused argument in any execution function (all interfaces, new return type or not). I don't know where it was supposed to be used but obviously we don't need it. Any insights? @josh146

Reply:
I do not recall why I did this :( Maybe I thought it would be important down the line, but didn't need it at the time

Reply:
Perhaps at some point the DeviceAPI would need to know the mode (@mlxd)

Reply:
If it isn't being used anywhere, I think killing it is fine. It can always be readded if/when needed. I want to move use to a different run-time execution config type for such options later, so we can keep such data in there as required.